kv/local_sender_test.go

```diff
@@ -43,6 +44,36 @@ func TestLocalSenderAddStore(t *testing.T) {
 	}
 }
 
+func TestLocalSenderRemoveStore(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	ls := NewLocalSender()
+
+	store := storage.Store{}
```

Comment:
ok, will do this. but a where is stated that zero is not valid store id? should there be a validation for this in "AddStore"?